src/util.js

```diff
@@ -1,33 +1,18 @@
 'use strict';
 
-function followPaginate(data) {
-  return new Promise((resolve, reject) => {
-    $.getJSON(data.url)
-    .then((response, status, xhr) => {
-      data.items = data.items.concat(response);
-      const link = xhr.getResponseHeader('Link');
-      let url = null;
-      if (link) {
-        const match = link.match(/<(.*?)>; rel="next"/);
-        url = match && match[1] ? match[1] : null;
-      }
-      resolve({ items: data.items, url: url });
-    })
-    .fail(reject);
-  });
-}
-
-function getAllItems(promise) {
-  return promise.then(followPaginate)
+function getAllItems(promise, followMethod, type) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/77d86c0cbe12e28b49d988b7fdf8d52ef89bd093/src/util.js#L3"></span>‘getAllItems’ is defined but never used.</td>
    </tr>
  </tbody>
</table>